### DISCUSSION TITLE
Spring-net for .net core

### DISCUSSION THREAD

**Discussion by jgermanos** [spring-projects/spring-net#226] [Ideas]
Hi is there support for .net core yet?   and if so which versions?

**Comment by lahma** [comment]
The latest version 3.0.0 on NuGet has netstandard2.0 support so it works on NET Core.

**Reply by jgermanos** [reply]
Thanks lahma.  Any plans for .net 5.0+ support in the future?

**Reply by swalters** [reply]
It supports .Net 5 and 6 today. I'm using in 6 in production

**Reply by TureyLab** [reply]
Is there a reason why Spring.Testing.Microsoft is not supporting netstandard2.0? How to do Unittesting?

**Reply by lahma** [reply]
> Is there a reason why Spring.Testing.Microsoft is not supporting netstandard2.0? How to do Unittesting?

@TureyLab maybe you can investigate if it's possible to get it working and send a PR if so?

**Comment by hantsy** [comment]
Please add the latest .net core SDK in the build pipeline, and adopt the latest .NET features as spring Java supports the latest Java features via multiple version build.

**Reply by lahma** [reply]
What does that even mean?

**Reply by hantsy** [reply]
1. Make sure it is compatible with the latest DotNET core.
2. Adopt the newest DOTNET features as soon as possible. Spring(Java) upgraded the baseline to Java 17 and supports Java 21 Virtual thread. When the user uses the newest version Java 21+, the virtual thread will be activated.

**Reply by lahma** [reply]
Please send pull requests for such features.
